File: KingdomCodeTool/wwwroot/Download/202410291354/Service/Implement/ThanhVienLichSuTruyCapService.cs
namespace Service.Implement
{
    public class ThanhVienLichSuTruyCapService : BaseService<ThanhVienLichSuTruyCap, IThanhVienLichSuTruyCapRepository>
    , IThanhVienLichSuTruyCapService
    {
    private readonly IThanhVienLichSuTruyCapRepository _ThanhVienLichSuTruyCapRepository;
    public ThanhVienLichSuTruyCapService(IThanhVienLichSuTruyCapRepository ThanhVienLichSuTruyCapRepository) : base(ThanhVienLichSuTruyCapRepository)
    {
    _ThanhVienLichSuTruyCapRepository = ThanhVienLichSuTruyCapRepository;
    }
    }
    }

File: KingdomCodeTool/wwwroot/Download/202410291354/Angular/shared/ThanhVienLichSuTruyCap.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { ThanhVienLichSuTruyCap } from 'src/app/shared/ThanhVienLichSuTruyCap.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class ThanhVienLichSuTruyCapService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "ThanhVienLichSuTruyCap";
    }
}

